.newt.yml

```diff
@@ -0,0 +1,5 @@
+tool-versions:
```

Comment:
This file should be removed. We do not use `newt` to build this project.

Reply:
I just removed it. It's fine without it.